pkgs/tools/system/rofi-systemd/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, rofi, systemd, coreutils, utillinux, gawk, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  name = "rofi-systemd-${version}";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "IvanMalison";
+    repo = "rofi-systemd";
+    rev = "v${version}";
+    sha256 = "1dbygq3qaj1f73hh3njdnmibq7vi6zbyzdc6c0j989c0r1ksv0zi";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -a rofi-systemd $out/bin/rofi-systemd
+  '';
+
+  wrapperPath = with stdenv.lib; makeBinPath [
+    rofi
+    coreutils
+    utillinux
+    gawk
+    systemd
+  ];
+
+  fixupPhase = ''
+    patchShebangs $out/bin
+
+    wrapProgram $out/bin/rofi-systemd --prefix PATH : "${wrapperPath}"
+  '';
```

Comment:
Using

```nix
{
  postFixup = ''
    wrapProgram $out/bin/rofi-systemd --prefix PATH : "${wrapperPath}"
  '';
}
```

instead of overriding the `fixupPhase` would be more elegant here.